src/App.js

```diff
@@ -31,26 +22,42 @@ function App() {
     const { lat, lon, image } = await getCityLatLongImageUrban(urban_api);
     const data = await getWeekForcast(lat, lon);
     console.log(`Base on Dropdown : `, data);
-  };
+  };*/
```

Comment:
We should never merge commented code into master.
If we don't need it - we should delete it, as with git it's easy to recover it if we need it in the future.
If it's there so you can remember later how to use it - put it in the documentation instead.